### PR TITLE
Fixed Night Mode in BlocksUI

### PR DESCRIPTION
**Pull request by Blacklion14**
Signed-off-by: Blacklion14 <rkush214@gmail.com>

## Changes

I changed ThIs [css](https://github.com/thanos-io/thanos/blob/main/pkg/ui/react-app/src/thanos/pages/blocks/blocks.module.css) file and added a z-index so that the blocks which were behind the container of bootstrap-dark have came to front

Also i changed this bootstrap-dark.css file a bit in order to make close Button in blocks details div to be visible clearly in dark mode. This change may not create any ambiguity because in general we will always willing to have a white contrast text colour inside a button in dark-mode . And this will change colour of only text inside button not the background colour. 
If some one wants to change colour to some other white contrast they can directly change it from here.

## Verification

I ran the UI and it is running clearly without any errors or bugs.

### CODE REVIEW

**Comment by ianbillett** [comment]
Nice work @Blacklion14 :muscle: Two things from me that will help reviewers with this PR:
* Does this PR close an issue?
* Could you provide some screen grabs of before and after your changes? That way we have a visual idea of what has changed.
Thanks!

**Comment by Blacklion14** [comment]
Hello @ianbillett 
1. I actually don't know if this closes a issue because i was running Blocks UI and i saw in night mode blocks were not visible.
So i changed the css a bit and asked @onprem on slack giving all the screenshots and changes and he guided me to create a PR.  

2. Previously on using Night mode in Block UI we were unable to see blocks clearly (as in Screenshot)
Screenshot from 2021-07-04 09-54-00

But as i changed css a bit which made it look like this 
Screenshot from 2021-07-05 21-43-13

Also i fixed the close button(in the section which appears by clicking on any block) which was not visible in darkmode.

These are the changes I made.

**Review by ianbillett — APPROVED** [review]
Awesome - nice work :muscle: Since you have been chatting with him, let's get @onprem 's review too before merging.

**Comment by Blacklion14** [comment]
Thank you @ianbillett for reviewing 🙇🙇
I am also waiting for @onprem 's review..

File: pkg/ui/react-app/src/themes/_bootstrap_dark.scss
Comment (onprem) [inline pkg/ui/react-app/src/themes/_bootstrap_dark.scss:42]:
```suggestion
body.bootstrap-dark button {
  color: $body-color;
}
```
We can use a variable for this instead of hardcoding white. I am not sure if the variable I suggested is the right one though.

**Review by onprem — APPROVED** [review]
LGTM. I have just one non-blocking comment. Thanks for the PR!

File: pkg/ui/react-app/src/themes/_bootstrap_dark.scss
Comment (onprem) [reply pkg/ui/react-app/src/themes/_bootstrap_dark.scss:42]: Another option that we can consider (if it works)
```suggestion
body.bootstrap-dark button{
  color: inherit;
}
```

I am just spitting this out of my head so we need to test first if this works

Comment (Blacklion14) [reply pkg/ui/react-app/src/themes/_bootstrap_dark.scss:42]: Got it !!
I will check this now.

Comment (Blacklion14) [reply pkg/ui/react-app/src/themes/_bootstrap_dark.scss:42]: I tested it out 
Your first suggestion works Nice , I did thought about that earlier that we shouldn't hard code but i didn't knew how to implement.
Thanks!!
Now what should i do @onprem ?? Create a new PR or edit this here

**Comment by GiedriusS** [comment]
@Blacklion14 please rebase on the latest `main` and don't forget to do `make assets` :muscle: let's get this into the next RC

**Comment by Blacklion14** [comment]
Sure @GiedriusS

**Comment by Blacklion14** [comment]
> @Blacklion14 please rebase on the latest `main` and don't forget to do `make assets`  let's get this into the next RC


Hello @GiedriusS 
I have tried to rebase my branch (fix-night-mode-bucketUI) with following commands
Screenshot from 2021-07-07 18-19-01

But i am facing merge conflicts issue in `pkg/ui/bindata.go` file.
Please guide me how can i resolve this issue and rebase my branch properly.

**Comment by GiedriusS** [comment]
> > @Blacklion14 please rebase on the latest `main` and don't forget to do `make assets`  let's get this into the next RC
> 
> Hello @GiedriusS
> I have tried to rebase my branch (fix-night-mode-bucketUI) with following commands
> Screenshot from 2021-07-07 18-19-01
> 
> But i am facing merge conflicts issue in `pkg/ui/bindata.go` file.
> Please guide me how can i resolve this issue and rebase my branch properly.

You could resolve this by accepting the incoming changes by opening that file with your favorite editor and then create another commit after running `make assets`

**Comment by Blacklion14** [comment]
> > > @Blacklion14 please rebase on the latest `main` and don't forget to do `make assets`  let's get this into the next RC
> > 
> > 
> > Hello @GiedriusS
> > I have tried to rebase my branch (fix-night-mode-bucketUI) with following commands
> > Screenshot from 2021-07-07 18-19-01
> > But i am facing merge conflicts issue in `pkg/ui/bindata.go` file.
> > Please guide me how can i resolve this issue and rebase my branch properly.
> 
> You could resolve this by accepting the incoming changes by opening that file with your favorite


> > > @Blacklion14 please rebase on the latest `main` and don't forget to do `make assets`  let's get this into the next RC
> > 
> > 
> > Hello @GiedriusS
> > I have tried to rebase my branch (fix-night-mode-bucketUI) with following commands
> > Screenshot from 2021-07-07 18-19-01
> > But i am facing merge conflicts issue in `pkg/ui/bindata.go` file.
> > Please guide me how can i resolve this issue and rebase my branch properly.
> 
> You could resolve this by accepting the incoming changes by opening that file with your favorite editor and then create another commit after running `make assets`

@GiedriusS I have done the changes you said . Can you please review....

**Comment by Blacklion14** [comment]
@GiedriusS I don't know why the above test is getting failed ....
Please review 🙇🙇

**Review by onprem — APPROVED** [review]
Awesome work. Thanks!

**Pull request merged**
